openmetrics/handler.go

```diff
@@ -0,0 +1,46 @@
+package openmetrics
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/bsm/pgpq"
+)
+
+// Client defines pgpq client (only needed methods).
+type Client interface {
+	Stats(context.Context) ([]*pgpq.Stat, error)
+}
+
+// NewHandler constructs a new handler to serve queue metrics in OpenMetrics format.
+func NewHandler(client Client) http.Handler {
```

Comment:
As dumb as it can possibly be. No need for extra deps or caching here, IMO.

But please, pay attention to used metric type - all good, "gauge"-s are fine?

Reply:
i.e. 

```
deduped := make(map[string]struct{}, len(namespaces)+1)
deduped[""] = struct{}{}
for _, ns := range namespaces {
  deduped[ns] = struct{}{}
}
```